install/rc.yaml

```diff
@@ -22,8 +22,8 @@ spec:
         emptyDir: {}
       containers:
       - name: brokersdk
-        image: brokersdk
-        imagePullPolicy: Never
+        image: maleck13/service-broker-mongo:1.4
```

Comment:
looks like you caught a personal change here.

i'm fine w/ the change to IfNotPresent on the pull policy though (but i believe that's the default, so you can just remove it entirely)